index.js

```diff
@@ -33,7 +33,7 @@ class Action {
             let addSourceCmd;
             if (this.nugetSource.startsWith(`https://nuget.pkg.github.com/`)) {
                 this.sourceType = "GPR"
-                addSourceCmd = `dotnet nuget add source ${this.nugetSource}/index.json --name=${(this.sourceName)} --username=${this.githubUser} --password=${this.nugetKey} --store-password-in-clear-text`
+                addSourceCmd = `dotnet nuget add source ${this.nugetSource}/${this.githubUser}/index.json --name=${(this.sourceName)} --username=${this.githubUser} --password=${this.nugetKey} --store-password-in-clear-text`
```

Comment:
This is a breaking change that will cause failures in existing workflows, sorry not going to merge it. 

It also won't let people use organization repos, and that's the main reason why `nugetSource` parameter looks like that: it suggests that you will use string like `https://nuget.pkg.github.com/orgname` or `https://nuget.pkg.github.com/username` depending on what type of repository you have.

Reply:
Agreed on that, looks like my bad

I suppose that current users of this action are just passing full link with a namespace included, that's why it's working for them.
I might be passing 'https://nuget.pkg.github.com/NAMESPACE' however I've tried to pass just 'https://nuget.pkg.github.com/' and it didn't work for me
So in my case the action have tried to send a GET to 'https://nuget.pkg.github.com/index.json' and failed for me as this is an incorrect url.